Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift

```diff
@@ -54,5 +66,13 @@ class MainDetailView: UIView {
             }
         }
     }
+    
+    @IBAction func onInstallButtonTapped(_ sender: Any) {
+        guard let iOSPackageName = showcaseApp?.iosPackageName, let url = URL(string: iOSPackageName) else {
```

Comment:
All this should be in the presenter and should be tested. you should ask the presenter to open the Url

Reply:
Ok i will update the PR with tests and move code to the presenter.